src/transformers/models/resnet/configuration_resnet.py

```diff
@@ -89,3 +95,20 @@ def __init__(
         self.layer_type = layer_type
         self.hidden_act = hidden_act
         self.downsample_in_first_stage = downsample_in_first_stage
+
+
+class ResNetOnnxConfig(OnnxConfig):
+
+    torch_onnx_minimum_version = version.parse("1.11")
+
+    @property
+    def inputs(self) -> Mapping[str, Mapping[int, str]]:
+        return OrderedDict(
+            [
+                ("pixel_values", {0: "batch", 1: "sequence"}),
+            ]
+        )
+
+    @property
+    def atol_for_validation(self) -> float:
+        return 1e-4
```

Comment:
Hey @chamidullinr nice PR, did you try the conversion config with the default `--atol=1e-5` before putting it to `1e-4` ?

If you are okay you can push a ResNet converted model to the [ONNXConfig for all](https://huggingface.co/OWG) organisation, it would be awesome! :hugs:

Reply:
I pushed a `ResNet-50` ONNX converted model. And I set `--atol=1e-3` because some tests were failing.